mate-settings-daemon/main.c

```diff
@@ -32,9 +32,6 @@
 #include <glib/gstdio.h>
 #include <gtk/gtk.h>
 
```

Comment:
missing include `#include <gio/gio.h>`

Reply:
test:
```
$ cat test2.sh 
declare -i NUM
for FILE in `git grep g_dbus_proxy -- name ':*.c' | cut -d':' -f1 | sort -u`; do
  NUM=`grep -F "#include <gio/gio.h>" $FILE | wc -l`
  echo -ne "$FILE"
  if [ $NUM -gt 0 ]
  then
    echo " [OK] ($NUM)"
  else
    echo " [FAIL]"
  fi
done
$ ./test2.sh 
mate-settings-daemon/main.c [FAIL]
plugins/a11y-keyboard/msd-a11y-preferences-dialog.c [OK] (1)
plugins/background/msd-background-manager.c [OK] (1)
plugins/media-keys/msd-media-keys-manager.c [OK] (1)
plugins/mpris/msd-mpris-manager.c [FAIL]
plugins/rfkill/msd-rfkill-manager.c [OK] (1)
plugins/smartcard/msd-smartcard-plugin.c [OK] (1)
```

Reply:
```
$ grep gtk/gtk.h plugins/mpris/msd-mpris-manager.c
#include <gtk/gtk.h>
$ grep gio /usr/lib/pkgconfig/gtk+-3.0.pc 
Requires: gdk-3.0 atk >= 2.15.1  cairo >= 1.14.0 cairo-gobject >= 1.14.0 gdk-pixbuf-2.0 >= 2.30.0 gio-2.0 >= 2.57.2
Requires.private: atk atk-bridge-2.0 wayland-client >= 1.14.91 wayland-protocols >= 1.14 xkbcommon >= 0.2.0 wayland-cursor >= 1.14.91 wayland-egl epoxy >= 1.4 fribidi >= 0.19.7 pangoft2 gio-unix-2.0 >= 2.57.2
```

Reply:
pkg-config's Requires field is a list of packages required by this package (run time), it has nothing to do with adding or not an include (build time). In addition, the Cflags field helps preprocessor to find the file included, but it must be included to the code.

Reply:
> The recommended way of using GLib has always been to only include the toplevel headers glib.h, glib-object.h, gio.h ...
> Still, there are some exceptions; these headers have to be included separately: gmodule.h, glib-unix.h, glib/gi18n-lib.h or glib/gi18n.h.

https://developer.gnome.org/glib/stable/glib-compiling.html